modules/nf-core/seqfu/check/main.nf

```diff
@@ -0,0 +1,56 @@
+process SEQFU_CHECK {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqfu:1.22.3--hc29b5fc_1':
+        'biocontainers/seqfu:1.22.3--hc29b5fc_1' }"
+
+    input:
+
+    tuple val(meta), path(reads)
+
+    output:
+
+    tuple val(meta), path("*_check.tsv")    , emit: check
+    path "versions.yml"                     , emit: versions
+
+    when:                                  
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}_check"
+
+    """
+    if [ -d "${reads}" ]; then
```

Comment:
Can we use a Groovy function to check if this is a directory?
If nothing else, make a temporary variable to pass, so you don't have to type out `seqfu check ....` twice.